test/models/cache_test.rb

```diff
@@ -78,47 +78,73 @@ def test_changing_observation_name
   # changed in all of a genus's subtaxa (and that updated_at not changed).
   def test_propagate_classification
     name = names(:agaricus)
+    saved_obs_updated_ats =
+      Observation.where(Observation[:text_name].matches("Agaricus%")).
+      map(&:updated_at)
     new_classification = names(:peltigera).classification
+
     name.update(classification: new_classification)
     name.propagate_classification
+
     Observation.where(
       Observation[:text_name].matches("Agaricus%")
     ).each do |obs|
       assert_equal(new_classification, obs.classification)
-      assert_operator(obs.updated_at, :<, 1.minute.ago)
     end
+    assert_equal(
+      saved_obs_updated_ats,
+      Observation.where(Observation[:text_name].matches("Agaricus%")).
+        map(&:updated_at)
+    )
```

Comment:
Oh wow, that's actually really clean.  I was scared to try this approach because I couldn't figure out how to do so easily.  This works nicely.